redis/cache.go

```diff
@@ -0,0 +1,27 @@
+package redis
+
+import (
+	"github.com/go-redis/cache"
+	"gopkg.in/vmihailenco/msgpack.v2"
+)
+
+// RedisCache is a *cache.Codec that can be set to the same cache you've created
+var RedisCache *cache.Codec
+
+// NewStandardRedisCache generates a preconfigured redis cache according to our spec.
+func NewStandardRedisCache(redisHostEnvironmentVariableName string) (codec *cache.Codec, err error) {
+	client, err := NewStandardRedisClient(redisHostEnvironmentVariableName)
+	if err != nil {
+		return
+	}
+	codec = &cache.Codec{
+		Redis: client,
+		Marshal: func(v interface{}) ([]byte, error) {
```

Comment:
Can you just set this to:
```go
Marshal:   msgpack.Marshal,
Unmarshal: msgpack.Unmarshal, 
```
Since those functions match the signature required for this parameter, there is no need for the wrapper function